pkg/kubelet/cm/topologymanager/policy_none_test.go

```diff
@@ -40,28 +41,44 @@ func TestName(t *testing.T) {
 
 func TestPolicyNoneCanAdmitPodResult(t *testing.T) {
 	tcases := []struct {
-		name     string
-		hint     TopologyHint
-		expected bool
+		name           string
+		providersHints []map[string][]TopologyHint
+		expectedHint   TopologyHint
+		expectedAdmit  lifecycle.PodAdmitResult
 	}{
 		{
-			name:     "Preferred is set to false in topology hints",
-			hint:     TopologyHint{nil, false},
-			expected: true,
+			name:           "merged empty providers hints",
+			providersHints: []map[string][]TopologyHint{},
+			expectedHint:   TopologyHint{},
+			expectedAdmit:  lifecycle.PodAdmitResult{Admit: true},
+		},
+		{
+			name: "merge with a single provider with a single preferred resource",
+			providersHints: []map[string][]TopologyHint{
+				{
+					"resource": {{NUMANodeAffinity: NewTestBitMask(0, 1), Preferred: true}},
+				},
+			},
+			expectedHint:  TopologyHint{},
+			expectedAdmit: lifecycle.PodAdmitResult{Admit: true},
```

Comment:
This seems to be a completely different set of tests now for this test case. Is there some way to keep the old tests and then add these as new ones?

Reply:
Looks like there was a mix-up here... `TestNoneCanAdmitPodResult` should have been left as is, and `TestPolicyMerge` should contain these new tests - will fix.